utils/gui.lua

```diff
@@ -18,6 +18,8 @@ local Gui = {}
 local data = {}
 local element_map = {}
 
+Gui.tag = '__@level-RedMew__'
```

Comment:
I'd call this something really specific like:

```suggestion
Gui.event_handler_tag = '__@level-RedMew__event_handler_tag'
```

And give tags precedence over names for events. I think in an ideal would we would convert all events that are using names to tags, but I can see that would be a lot of effort. I think it could be good to be able to use names for someone over than events, e.g. debugging or to lookup child within parents, and that is less error prone if tags take precedence.

Reply:
I'm not too opposed to converting all names to tags. It's a bit of effort yes but it only requires to move the uid to the key below of the guielem definition, and it is a non-breaking change (would not be possible to convert all tags to names instead).

The tag string could be a SHA 4096 key for all I care, but I'd like to keep the key exported by the module short because it makes the gui code, which is already quite erratic, a lil bit easier to read if keys are few chars long instead of 20+

Reply:
Regarding the tag name, I was mostly thinking what if we added another tag in the future. I guess we could solve that problem later as we can always change the names.

Reply:
Changed to `Gui.event_tag`